.github/workflows/update-openapi.yml

```
@@ -0,0 +1,159 @@
+      - name: Build project with updated spec
+        if: steps.fetch-spec.outputs.has_changes == 'true' && steps.check-prs.outputs.create_pr == 'true'
+        run: |
+          # Clean and rebuild
+          rm -rf ./node_modules
+          pnpm install
+          pnpm build:pre
+          pnpm run build
+      
+      - name: Upload artifacts
+        if: steps.fetch-spec.outputs.has_changes == 'true' && steps.check-prs.outputs.create_pr == 'true'
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)